pkg/feed/opml.go

```diff
@@ -0,0 +1,44 @@
+package feed
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mxpv/podsync/pkg/config"
+	"github.com/mxpv/podsync/pkg/db"
+	"github.com/mxpv/podsync/pkg/fs"
+	"github.com/pkg/errors"
+)
+
+func BuildOPML(ctx context.Context, config *config.Config, db db.Storage, fs fs.Storage) (string, error) {
+
+	xmlString := `<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<opml version="1.0">
```

Comment:
I'd prefer to use existing package instead of string manipulations. For instance, it looks like `go-opml` does the job: https://github.com/gilliek/go-opml/blob/master/opml/opml.go

Reply:
I already had a look to the package you mentioned but it seems useful just for parsing OPML files not to create them. 
I've just refactored the code to use XML package, PR is updated.

Reply:
I think it is useful for serialization as well.
You can do
```go
f := OPML{}
// fill it up
return f.XML() // returns marshaled string
```
I'd prefer to avoid maintaining own structures.

Reply:
I'll have a look on this later. Let's merge this.